{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"connectorAuthenticationException"},{"kind":"text","text":"("},{"kind":"typeIdentifier","identifier":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/ConnectorAuthenticationException","preciseIdentifier":"s:10AWSAppflow32ConnectorAuthenticationExceptionV","text":"ConnectorAuthenticationException"},{"kind":"text","text":")"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/awsappflow\/registerconnectoroutputerror\/connectorauthenticationexception(_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/RegisterConnectorOutputError\/connectorAuthenticationException(_:)","interfaceLanguage":"swift"},"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"connectorAuthenticationException"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"ConnectorAuthenticationException","preciseIdentifier":"s:10AWSAppflow32ConnectorAuthenticationExceptionV"},{"kind":"text","text":")"}],"title":"RegisterConnectorOutputError.connectorAuthenticationException(_:)","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:10AWSAppflow28RegisterConnectorOutputErrorO32connectorAuthenticationExceptionyAcA0cgH0VcACmF","modules":[{"name":"AWSAppflow"}]},"hierarchy":{"paths":[["doc:\/\/AWSAppflow\/documentation\/AWSAppflow","doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/RegisterConnectorOutputError"]]},"references":{"doc://AWSAppflow/documentation/AWSAppflow/RegisterConnectorOutputError/connectorAuthenticationException(_:)":{"role":"symbol","title":"RegisterConnectorOutputError.connectorAuthenticationException(_:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"connectorAuthenticationException"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"ConnectorAuthenticationException","preciseIdentifier":"s:10AWSAppflow32ConnectorAuthenticationExceptionV"},{"kind":"text","text":")"}],"abstract":[],"identifier":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/RegisterConnectorOutputError\/connectorAuthenticationException(_:)","kind":"symbol","type":"topic","url":"\/documentation\/awsappflow\/registerconnectoroutputerror\/connectorauthenticationexception(_:)"},"doc://AWSAppflow/documentation/AWSAppflow/RegisterConnectorOutputError":{"role":"symbol","title":"RegisterConnectorOutputError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"RegisterConnectorOutputError"}],"abstract":[],"identifier":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/RegisterConnectorOutputError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"RegisterConnectorOutputError"}],"url":"\/documentation\/awsappflow\/registerconnectoroutputerror"},"doc://AWSAppflow/documentation/AWSAppflow":{"role":"collection","title":"AWSAppflow","abstract":[],"identifier":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow","kind":"symbol","type":"topic","url":"\/documentation\/awsappflow"},"doc://AWSAppflow/documentation/AWSAppflow/ConnectorAuthenticationException":{"role":"symbol","title":"ConnectorAuthenticationException","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"ConnectorAuthenticationException"}],"abstract":[{"type":"text","text":"An error occurred when authenticating with the connector endpoint."}],"identifier":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/ConnectorAuthenticationException","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ConnectorAuthenticationException"}],"url":"\/documentation\/awsappflow\/connectorauthenticationexception"}}}